### PR TITLE
create css from env variables

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: freebasics/templates/custom_css.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<style type="text/css">
```
Comment (miltontony) [inline freebasics/templates/custom_css.html:1]: @codiebeulaine this style comes from [this template](https://github.com/projectmushroom/fb-admin/blob/master/dist/css/skins/template-base.css#L156-L175), you can see the preview [here](http://projectmushroom.com/fb_admin_v1.8/export.html)

**Comment by miltontony** [comment]
TODO: the env variable names have changed - the `custom_css.html` template needs to be updated to fill in the values from the context

File: freebasics/settings/base.py [hunk 2]
```diff
@@ -212,6 +212,19 @@
 
 ENABLE_SSO = False
 
+CUSTOM_CSS_BASE_BACKGROUND_COLOR = environ.get(
+    'CUSTOM_BODY_BACKGROUND_COLOR', "#ffffff")
```
Comment (miltontony) [inline freebasics/settings/base.py]: The `env` variable here is `CUSTOM_CSS_BASE_BACKGROUND_COLOR`

Comment (miltontony) [inline freebasics/settings/base.py]: And same for the rest below - they should match the env variables we're creating in `mc2-freebasics`

**Comment by miltontony** [comment]
Also, test please

**Comment by codiebeulaine** [comment]
@miltontony @saeedmb8 ready for review

**Comment by miltontony** [comment]
:+1: :cake:

**Pull request merged**